app/standard_operations.py

```diff
@@ -21,57 +21,15 @@ def get_operations(self):
         response_content.raise_for_status()
         response_dict = response_content.json()
 
-        operations = []
-        for op in response_dict["$defs"]:
-            operations.append({
-                "source_url": source_url,
-                "url": url,
-                "operations": operations,
-                "version": version,
-                "title": title,
-                "x-maturity": x_maturity
-            })
-
-        return endpoints
-
-
-def main():
-    """Run CLI."""
-    import argparse
-    import json
-
-    argparser = argparse.ArgumentParser(
-        description="CLI testing of the ResponseCache class"
-    )
-    argparser.add_argument(
-        "--get_trapi_endpoints",
-        action="count",
-        help="Get a list of TRAPI endpoints",
-    )
-    argparser.add_argument(
-        "--get_operations_endpoints",
-        action="count",
-        help="Get a list of TRAPI endpoints that support operations",
-    )
-    args = argparser.parse_args()
-
-    if (
-        args.get_trapi_endpoints is None
-        and args.get_operations_endpoints is None
-    ):
-        argparser.print_help()
-        return
-
-    smartapi = SmartAPI()
-
-    if args.get_trapi_endpoints:
-        endpoints = smartapi.get_trapi_endpoints()
-        print(json.dumps(endpoints, sort_keys=True, indent=2))
-
-    if args.get_operations_endpoints:
-        endpoints = smartapi.get_operations_endpoints()
-        print(json.dumps(endpoints, sort_keys=True, indent=2))
-
-
-if __name__ == "__main__":
-    main()
+        operations = {}
+        for op, props in response_dict["$defs"].items():
+            
+            operations[op] = {
+                "id": op,
```

Comment:
So this won't actually work. the ID isn't the the same thing as the name for the operations. If you take a look at the schema file, you'll see that the name will be, for instance, "OperationAnnotateNodes" but the id will be "annotate_nodes". You actually need to pull the id from there.

Reply:
Good point. Update pushed.